client/src/client-only-routes/ShowCertification.js

```diff
@@ -113,6 +113,8 @@ class ShowCertification extends Component {
     if (
       !isDonationDisplayed &&
       userComplete &&
+      signedInUserName &&
+      username &&
```

Comment:
```suggestion
```
I think this can go, because the next line makes sure they're the same, so we don't need to check both.